spec/requests/api/v1/sessions/create_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+describe 'POST api/v1/users/sign_in', type: :request do
+  let(:password) { 'password' }
+  let(:token) do
+    {
+      '70crCAAYmNP1xLkKKM09zA' =>
+      {
+        'token' => '$2a$10$mSeRnpVMaaegCpn3AhORGe5wajFhgMoBjGIrMwq4Qq2mP6f/OHu1y',
+        'expiry' => 153_574_356_4
+      }
+    }
+  end
```

Comment:
Try removing this and the tokens from the line below and check if the sign in works. I don't think we need this.